grabdish/undeploy.sh

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2021 Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-MS="frontend-helidon order-helidon supplier-helidon-se inventory-helidon"
+MS="frontend-helidon order-helidon supplier-helidon-se inventory-helidon foodwinepairing-python"
```

Comment:
as mentioned in deploy.sh comment let's take this out and have user do it explicitly/separately